### PR TITLE
Add Flake8 to the CI

### PR DESCRIPTION
**Pull request by Fokko**
- Rename `next` to `__next__`
- Fix indentation
- Fix faulty string formatting

### CODE REVIEW

**Comment by rdblue** [comment]
@TGooch44, can you help take a look?

Looks like this adds a static analysis tool that caught a few inconsistencies.

**Comment by TGooch44** [comment]
Looks great, will take me a bit to go through it all, but I'm all for it.

**Comment by rdblue** [comment]
Looks like this probably can't be merged until the other PRs are finished first. Is it possible to separate this out with just the changes to apply the style checker?

**Comment by Fokko** [comment]
Thanks. I've added it to `tox` now, so we can figure out if we want to use pre-commit later on :)

File: python/tests/api/expressions/conftest.py [hunk 1]
```diff
@@ -90,7 +90,7 @@ def __init__(self, message):
 
         def predicate(self, pred):
             if isinstance(pred, UnboundPredicate):
-                assert False
+                raise AssertionError("Predicate should be an UnboundPredicate")
```
Comment (rdblue) [inline python/tests/api/expressions/conftest.py:93]: This should _not_ be an `UnboundPredicate`.

Comment (rdblue) [reply python/tests/api/expressions/conftest.py]: I updated the error message before merging.

**Comment by rdblue** [comment]
Looks good to me. Thanks for separating this out and using Tox!

**Comment by Fokko** [comment]
Thanks! I find tox a bit more cumbersome, but introducing a new tool might not be worth the added complexity in the CI pipeline.

**Pull request merged**